### PR TITLE
dumpling: refined as per cn doc

### PR DESCRIPTION
**Pull request by hfxsd**
### What is changed, added or deleted? (Required)
Refined the structure and polished some descriptions.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [x] v6.6 (TiDB 6.6 versions)
- [x] v6.5 (TiDB 6.5 versions)
- [x] v6.4 (TiDB 6.4 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [x] v5.4 (TiDB 5.4 versions)
- [x] v5.3 (TiDB 5.3 versions)
- [x] v5.2 (TiDB 5.2 versions)
- [x] v5.1 (TiDB 5.1 versions)
- [x] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:https://github.com/pingcap/docs-cn/pull/7143
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: dumpling-overview.md [hunk 1]
```diff
@@ -61,7 +30,36 @@ In the above commands, you need to modify `~/.bash_profile` to the path of your
...
+- For real-time backups of incremental data, refer to [TiCDC](/ticdc/ticdc-overview.md).
+- All exported data can be imported back to TiDB using [TiDB Lightning](/tidb-lightning/tidb-lightning-overview.md).
+
+</CustomContent>
+
+> **Note:**
+>
+> PingCAP previously maintained a fork of the [mydumper project](https://github.com/maxbube/mydumper) with enhancements specific to TiDB. This fork has since been replaced by [Dumpling](/dumpling-overview.md), which has been rewritten in Go, and supports more optimizations that are specific to TiDB. It is strongly recommended that you use Dumpling instead of mydumper.
+>
+> For the overview of Mydumper, refer to [v4.0 Mydumper documentation](https://docs.pingcap.com/tidb/v4.0/backup-and-restore-using-mydumper-lightning).
```
Comment (ran-huang) [inline dumpling-overview.md:51]:
```suggestion
> For more information on Mydumper, refer to [v4.0 Mydumper documentation](https://docs.pingcap.com/tidb/v4.0/backup-and-restore-using-mydumper-lightning).
```

File: dumpling-overview.md [hunk 2]
```diff
@@ -61,7 +30,36 @@ In the above commands, you need to modify `~/.bash_profile` to the path of your
...
+
```
Comment (ran-huang) [inline dumpling-overview.md:52]:
```suggestion

Compared to Mydumper, Dumpling has the following improvements:

```

File: dumpling-overview.md [hunk 3]
```diff
@@ -61,7 +30,36 @@ In the above commands, you need to modify `~/.bash_profile` to the path of your
...
+- Support exporting data to Amazon S3 cloud storage.
+- More optimizations are made for TiDB:
+    - Support configuring the memory limit of a single TiDB SQL statement.
+    - Support automatic adjustment of TiDB GC time for TiDB v4.0.0 and later versions.
+    - Use TiDB's hidden column `_tidb_rowid` to optimize the performance of concurrent data export from a single table.
+    - For TiDB, you can set the value of [`tidb_snapshot`](/read-historical-data.md#how-tidb-reads-data-from-history-versions) to specify the time point of the data backup. This ensures the consistency of the backup, instead of using `FLUSH TABLES WITH READ LOCK` to ensure the consistency.
+
+## Export data from TiDB or MySQL
 
 ### Required privileges
```
Comment (ran-huang) [inline dumpling-overview.md:65]: Please add [this paragraph](https://github.com/pingcap/docs-cn/pull/7143/files#diff-d167a2ebb86d802c7a7279da3de5de87e8b616ed0a27755ef0027a8db570c147R61) to line 73:

```
本文假设在 `127.0.0.1:4000` 有一个 TiDB 实例，并且这个 TiDB 实例中有无密码的 root 用户。
```

File: dumpling-overview.md [hunk 4]
```diff
@@ -225,11 +223,11 @@ You can use the `--compress <format>` option to compress the CSV and SQL data an
```
Comment (ran-huang) [inline dumpling-overview.md:223]: Two notes are missing from line 152:

```
> - `--sql` 选项仅仅可用于导出 CSV 的场景。
> - 该命令将在要导出的所有表上执行 `select * from <table-name> where id < 100` 语句。如果部分表没有指定的字段，那么导出会失败。
```

File: dumpling-overview.md [hunk 5]
```diff
@@ -379,7 +377,7 @@ When Dumpling is exporting a large single table from TiDB, Out of Memory (OOM) m
 + Reduce the value of `--tidb-mem-quota-query` to `8589934592` (8 GB) or lower. `--tidb-mem-quota-query` controls the memory usage of a single query statement in TiDB.
 + Adjust the `--params "tidb_distsql_scan_concurrency=5"` parameter. [`tidb_distsql_scan_concurrency`](/system-variables.md#tidb_distsql_scan_concurrency) is a session variable which controls the concurrency of the scan operations in TiDB.
 
-### TiDB GC settings when exporting a large volume of data (more than 1 TB)
+### Set TiDB GC when exporting a large volume of data (more than 1 TB)
 
 When exporting data from TiDB (more than 1 TB), if the TiDB version is later than or equal to v4.0.0 and Dumpling can access the PD address of the TiDB cluster, Dumpling automatically extends the GC time without affecting the original cluster.
```
Comment (ran-huang) [inline dumpling-overview.md:383]: Line 400~410 were removed from the [Chinese version](https://github.com/pingcap/docs-cn/pull/7143/files#diff-d167a2ebb86d802c7a7279da3de5de87e8b616ed0a27755ef0027a8db570c147L332). Please also delete them from English.

File: dumpling-overview.md (hunk 4 above)
Comment (hfxsd) [reply dumpling-overview.md:225]: 在 L142 行里有的

File: dumpling-overview.md (hunk 5 above)
Comment (hfxsd) [reply dumpling-overview.md:385]: OK done

**Review by ran-huang — APPROVED** [review]

**Pull request merged**